### PR TITLE
Renamed u8timeseries to darts

### PR DESCRIPTION
**Pull request by hrzn**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -142,7 +142,7 @@ jobs:
       - name: "4. Build and publish docker to docker hub"
         uses: elgohr/Publish-Docker-Github-Action@2.14
         with:
-          name: unit8/u8timeseries
+          name: unit8/darts
```
Comment (TheMP) [inline .github/workflows/release.yml:145]: @endrjuskr do we need to create something new on Dockerhub for that to work?

**Comment by TheMP** [comment]
@endrjuskr @hrzn is it going to work with Sphinx?

File: .github/workflows/release.yml (hunk 1 above)
Comment (endrjuskr) [reply .github/workflows/release.yml:145]: yes I need, will take care of it

**Comment by endrjuskr** [comment]
So we our library on pypi is called u8darts, but package is called darts. Will it create conflict?

File: .github/workflows/release.yml (hunk 1 above)
Comment (endrjuskr) [reply .github/workflows/release.yml:145]: done :)

File: README.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
-# u8timeseries: Easy manipulation and forecasting of time series
+# darts: Easy manipulation and forecasting of time series
 
-![u8timeseries](https://github.com/unit8co/u8timeseries/raw/develop/static/images/logo-72dpi.png "u8timeseries") 
+![darts](https://github.com/unit8co/darts/raw/develop/static/images/logo-72dpi.png "darts") 
 
 ---
-[![PyPI version](https://badge.fury.io/py/u8timeseries.svg)](https://badge.fury.io/py/u8timeseries)
-![GitHub Workflow Status](https://img.shields.io/github/workflow/status/unit8co/u8timeseries/develop)
+[![PyPI version](https://badge.fury.io/py/darts.svg)](https://badge.fury.io/py/darts)
```
Comment (endrjuskr) [inline README.md:6]: IMO it should be u8darts

File: README.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/unit8co/darts/develop)
 ![Supported versions](https://img.shields.io/badge/python-3.6+-blue.svg)
-![Docker Image Version (latest by date)](https://img.shields.io/docker/v/unit8/u8timeseries?label=docker&sort=date)
-![PyPI - Downloads](https://img.shields.io/pypi/dm/u8timeseries)
-![GitHub Release Date](https://img.shields.io/github/release-date/unit8co/u8timeseries)
+![Docker Image Version (latest by date)](https://img.shields.io/docker/v/unit8/darts?label=docker&sort=date)
+![PyPI - Downloads](https://img.shields.io/pypi/dm/darts)
```
Comment (endrjuskr) [inline README.md:10]: same here `u8darts`

File: README.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
+![GitHub Release Date](https://img.shields.io/github/release-date/unit8co/darts)
```
Comment (endrjuskr) [inline README.md:11]: We need to change repo name as well

File: darts/__init__.py [hunk 5]
```diff
@@ -1,5 +1,5 @@
 """
-u8timeseries
+darts
 ------------
```
Comment (endrjuskr) [inline darts/__init__.py:3]: can we match line length to title?

File: darts/models/theta.py [hunk 6]
```diff
@@ -9,7 +9,7 @@
 import numpy as np
 import statsmodels.tsa.holtwinters as hw
 
-from u8timeseries.utils.statistics import check_seasonality, extract_trend_and_seasonality, remove_seasonality
+from darts.utils.statistics import check_seasonality, extract_trend_and_seasonality, remove_seasonality
```
Comment (endrjuskr) [inline darts/models/theta.py:12]: did we want to unify imports?

**Comment by hrzn** [comment]
Normally having the package in pypi called `u8darts` shouldn't cause conflicts, but we actually might be able to get the name `darts` so let's wait a week:
image

File: README.md (hunk 4 above)
Comment (TheMP) [reply README.md:11]: yep, the question is before or after merging - in both cases I think something will break ;]

File: darts/models/theta.py (hunk 6 above)
Comment (pennfranc) [reply darts/models/theta.py:12]: Yeah I think the plan was to make them all relative, if I'm not mistaken.

File: README.md (hunk 3 above)
Comment (hrzn) [reply README.md]: done. The other badges should display correctly once we have renamed the github repo.

File: README.md (hunk 4 above)
Comment (hrzn) [reply README.md:11]: let's try to change the repo name after merging

File: .github/workflows/release.yml [hunk 7]
```diff
@@ -1,4 +1,4 @@
-name: u8timeseries release workflow
+name: darts release workflow
```
Comment (endrjuskr) [inline .github/workflows/release.yml:1]: We might need to change requirements for PR

File: README.md [hunk 8]
```diff
@@ -1,16 +1,16 @@
-# u8timeseries: Easy manipulation and forecasting of time series
+# darts: Easy manipulation and forecasting of time series
 
-![u8timeseries](https://github.com/unit8co/u8timeseries/raw/develop/static/images/logo-72dpi.png "u8timeseries") 
+![darts](https://github.com/unit8co/darts/raw/develop/static/images/logo-72dpi.png "darts")
```
Comment (endrjuskr) [inline README.md:3]: We will need to change repository to show image

File: conda_recipe/darts/meta.yaml [hunk 9]
```diff
@@ -1,12 +1,12 @@
 {% set data = load_setup_py_data() %}
 
 package:
-  name: "u8timeseries"
+  name: "darts"
```
Comment (endrjuskr) [inline conda_recipe/darts/meta.yaml:4]: Are we publishing to conda under name darts?

File: README.md (hunk 2 above)
Comment (hrzn) [reply README.md]: done

File: darts/__init__.py (hunk 5 above)
Comment (hrzn) [reply darts/__init__.py]: done

File: darts/models/theta.py (hunk 6 above)
Comment (hrzn) [reply darts/models/theta.py]: done

File: README.md (hunk 8 above)
Comment (hrzn) [reply README.md:3]: yes, will do after merging

File: conda_recipe/darts/meta.yaml (hunk 9 above)
Comment (hrzn) [reply conda_recipe/darts/meta.yaml:4]: I don't think it's planned yet.

**Pull request merged**